build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.2.60'
```

Comment:
can you remove the file totally please ? it's not needed, AS created me at some point I didn't notice ^^ 
instead you can change the one under the example app of Crashlytics.

By changing that it will force all people with an older version to upgrade there kotlin version. Can you add on the Readme that Kotlin version should be at least 1.2.61 or more.

Reply:
Thanks :) also the gradle version as you updated it too :)